src/main/java/com/bravo/user/utility/ValidatorUtil.java

```diff
@@ -46,6 +46,14 @@ else if(value instanceof String){
     }
     return isValid;
   }
+  
+  public static boolean isEmailValid(final String string){
+    return string != null && string.matches("^([_a-zA-Z0-9-]+(\\.[_a-zA-Z0-9-]+)*@[a-zA-Z0-9-]+(\\.[a-zA-Z0-9-]+)*(\\.[a-zA-Z]{1,6}))?$");
```

Comment:
string is a hard to follow name here.  The method says to pass in an email so it's followable, inside of the method, we can't tell that string is and email.

Reply:
Again...agree here.   The variable should be more descriptive probably using "email" for the string variable name.